packages/preview/sano-presentation-template/1.0.0/template/main.typ

```diff
@@ -0,0 +1,58 @@
+#import "@preview/sano-presentation-template:1.0.0": *
+
+#show: sano.with(
+  config-info(
+    title: [Title for your awesome presentation in Typst],
+    author: [Your Name],
+  ),
+)
+
+#title-slide[
+Write a small and descriptive description for your presentation. You can have more than a sentence and it still looks awesome.
+]
+
+= Show your heading text for new sections in big and coloured format.
+
+== Grids to split your content
+
+#grid(columns: 2, gutter: 2em, [
+  - This slide type is ideal for comparing content.
+  - For example, a column can contain an image or chart.
+  - Thanks to Touying's package integration, we can, for example, reveal Fletcher diagrams piece by piece.
+  - Just like in this graph, node "d" has now been revealed.
+], [
+  #{
+    import "@preview/fletcher:0.5.2" as fletcher: node, edge
```

Comment:
You should update fletcher to the latest version to get rid of the deprecation warnings.